src/main/java/mif/vu/ikeea/Manager/LearningDayManager.java

```diff
@@ -37,6 +38,13 @@
     @Transactional
     public LearningDay create(LearningDayRequest learningDayRequest) {
         ApplicationUser user = userService.loadById(learningDayRequest.getUserId());
+
+        if(user.getRestrictionDays()-1 < 0){
```

Comment:
Siūlau pasidaryti ApplicationUser entityje metodus incrementRectrictionDays and decrementRestrictionDays ir juos naudoti, nes jeigu kada prireiks keisti dienas tai reiks eit per visą kodą ir ieškot :)